src/components/Header/Header.tsx

```diff
@@ -1,14 +1,27 @@
+import { useState } from 'react';
 import styles from './Header.module.css';
 import Logo from '../Logo';
 import NavBar from '../NavBar';
 import LogIn from '../LogIn';
+import LogInModal from '../LogInModal';
 
 function Header() {
+   const [isModalVisible, setIsModalVisible] = useState(false);
+
+   const handleClick = () => {
+      setIsModalVisible(true);
+   };
+
```

Comment:
```handleOpenModal```? Or even just ```openModal```

Reply:
To stick with the convention of onEvent - handleEvent